radixdlt-java/src/main/java/com/radixdlt/client/core/atoms/Atom.java

```diff
@@ -183,6 +200,12 @@ private static BigInteger ownedTokensToBigInteger(SpunParticle<OwnedTokensPartic
 		return (value.getSpin() == Spin.UP) ? bi : bi.negate();
 	}
 
+
+
+	public ImmutableMap<String, String> getMetaData() {
```

Comment:
As noted elsewhere, I would like to see this as a `Map<String, String>` so that Guava types are not leaked into the public interface.  I think this issue is more important to resolve here in the library.